torch/csrc/jit/codegen/cuda/lower_loops.h

```diff
@@ -40,7 +42,7 @@ class TORCH_CUDA_API LoopNestGenerator : public OptOutDispatch {
 
   // Predicates from ThreadPredicates that we will extend to reduction buffer
   // initialization
-  std::unordered_map<const TensorView*, Bool*>& thread_predicates_;
+  ThreadPredicateMap& thread_predicates_;
```

Comment:
const ?

Reply:
It can't have const as it can be modified in `LoopNestGenerator::initReduction`.